x-pack/plugins/infra/public/pages/logs/log_entry_categories/page.tsx

```diff
@@ -7,10 +7,22 @@
 
 import { EuiErrorBoundary } from '@elastic/eui';
 import React from 'react';
+import { useBreadcrumbs } from '../../../hooks/use_breadcrumbs';
 import { LogEntryCategoriesPageContent } from './page_content';
 import { LogEntryCategoriesPageProviders } from './page_providers';
+import { logCategoriesTitle } from '../../../../public/translations';
+import { LOGS_APP } from '../../../../common/constants';
 
 export const LogEntryCategoriesPage = () => {
+  useBreadcrumbs(
```

Comment:
Linking in with the feedback from above this would become (potentially) `useLogsBreadcrumbs`.